docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+services:
+  api:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    volumes:
+      - ./filedata:/data/
```

Comment:
Make this mapping not to the local filesystem but to a separate volume as you've done for redis. In this pull request `redis` service is not required.